wfc3tools/__init__.py

```diff
@@ -32,3 +34,12 @@
     teal.print_tasknames(__name__, os.path.dirname(__file__))
 except (ImportError, UnboundLocalError):
     print("Teal not installed, gui param editing disabled")
+
+if sys.version_info < (3, 6):
+    raise ImportError("wfc3tools supports Python versions 3.6 and above.")  # pragma: no cover
+
+try:
+    __version__ = get_distribution(__name__).version
```

Comment:
This is overkill. If setuptools_scm worked correctly, you can do `from .version import version as __version__` directly.

Reply:
@nden commented to say I should add __version__ in __init__

Reply:
i've remove the minimum version check in this file, since setup.cfg should be enforcing this (i think?). i also did not add a __version__ here because even without declaring it here, I am still able to access wfc3tools.__version__, so I assume this is done by setuptools